public/components/custom_panels/custom_panel_table.tsx

```diff
@@ -77,39 +78,32 @@ interface Props {
   loading: boolean;
   setBreadcrumbs: (newBreadcrumbs: ChromeBreadcrumb[]) => void;
   parentBreadcrumbs: EuiBreadcrumb[];
-  cloneCustomPanel: (newCustomPanelName: string, customPanelId: string) => void;
-  deleteCustomPanelList: (customPanelIdList: string[], toastMessage: string) => any;
   addSamplePanels: () => void;
 }
 
 export const CustomPanelTable = ({
   loading,
   setBreadcrumbs,
   parentBreadcrumbs,
-  cloneCustomPanel,
-  deleteCustomPanelList,
   addSamplePanels,
 }: Props) => {
   const customPanels = useSelector<CustomPanelType[]>(selectPanelList);
   const [isModalVisible, setIsModalVisible] = useState(false); // Modal Toggle
   const [modalLayout, setModalLayout] = useState(<EuiOverlayMask />); // Modal Layout
   const [isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
-  const [selectedCustomPanels, setselectedCustomPanels] = useState<CustomPanelType[]>([]);
+  const [selectedCustomPanels, setselectedCustomPanels$] = useState<CustomPanelType[]>([]);
```

Comment:
Not sure if the `$` is expected here. I see the below reference is `setselectedCustomPanels`

Reply:
There was debugging.  Will double-check.